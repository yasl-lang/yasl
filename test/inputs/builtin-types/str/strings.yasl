##123 .\n123 .\n123 .\n
# tests that strings are garbage collected properly

let x = 'x'

x = 'X'

x = '$'

# tests that string interpolation parses properly
const t = { .s: collections.set() }

echo "123 #{t.s->tostr()->ltrim('set(')->rtrim(")")}."

echo "#{'123'} ."

echo "#{"123"} ."

