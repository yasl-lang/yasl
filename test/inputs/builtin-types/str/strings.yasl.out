123 .
123 .
123 .
